CHANGELOG.rst

```diff
@@ -16,6 +16,13 @@ Subsections for each version can be one of the following;
 
 Each individual change should have a link to the pull request after the description of the change.
 
+1.1.2 (2024-02-07)
+------------------
+
+Added
+^^^^^
+- Updated NullIndicator to return int8 columns https://github.com/lvgig/tubular/pull/173
```

Comment:
please update to the format in rest of changelog (look at the raw markdown, I cant copy it in here without rendering messing with the code!)

Reply:
hopefully that's fixed here? https://github.com/lvgig/tubular/pull/173/commits/0651eafb7f6317751751354e9795cb7f2be06c57